web/yo/app/views/modifyTherapy.html

```diff
@@ -0,0 +1,38 @@
+<div class="modal-header">
+    <h4 class="modal-title">
+        Add Therapy</h4></div>
+<div class="modal-body">
+    <div class="input-group input-group-sm" style="width: 100%">
+        <div ng-repeat="element in therapy">
+            <div ng-click="addTherapy($index)" class="col-lg-10 col-md-10 col-sm-10 col-xm-10">
+                <tags-input ng-model="element.tags" display-property="drugName" replace-spaces-with-dashes="false" placeholder="  " add-from-autocomplete-only="true" on-tag-added="addDruginTherapy($tag, $index)" on-tag-removed="removeDruginTherapy($tag, $index)">
+                    <auto-complete source="loadDrugs($query)"
+                                   min-length="1"
+                                   load-on-focus="true"
+                                   load-on-empty="false"
+                                   template="drugsDropdown"></auto-complete>
+                </tags-input>
+            </div>
+            <button class="col-lg-1 col-md-1 col-sm-1 col-xm-1" ng-click="deleteTherapy($index)" style="background-color: white; border-color: white;"><i class="fa fa-remove" style="font-size:18px; margin-left:-10px; margin-top: 10px;"></i></button>
```

Comment:
Let's keep the style consistency, use the class='btn btn-default [maybe btn-sm?]'